etc/config/catchuparchiver_config.yaml

```diff
@@ -0,0 +1,41 @@
+# Configuration settings for CatchupArchiver
+ROOT:
+  REDIS_HOST: localhost
+  ARCHIVER_REDIS_DB: 14
```

Comment:
Might want to make this another DB number, unless you're sure it won't interfere with what Comcam is doing.

Reply:
I will update the db num.